bleachbit/FileUtilities.py

```diff
@@ -92,31 +92,39 @@ def __random_string(length):
 
 
 def bytes_to_human(bytes_i):
-    """Display a file size in human terms (megabytes, etc.) using SI standard"""
+    """Display a file size in human terms (megabytes, etc.) using preferred standard (SI or IEC)"""
 
-    storage_multipliers = {1000 ** 5: 'PB', 1000 ** 4: 'TB',
-                           1000 ** 3: 'GB', 1000 ** 2: 'MB', 1000: 'kB', 1: 'B'}
+    if bytes_i < 0:
+        return '-' + bytes_to_human(-bytes_i)
+
+    from Options import options
+    if options.get('units_iec'):
+        prefixes = ['','Ki','Mi','Gi','Ti','Pi']
+        base = 1024.0
+    else:
```

Comment:
Because this function can be called a thousand times (once for each file, like when cleaning localizations), I ran this benchmark in three scenarios
````timeit.timeit('bleachbit.FileUtilities.bytes_to_human(100000)', setup='import bleachbit.FileUtilities', number=100000)````

Original code: 4.7 seconds
New code: 9.2 seconds
New code that replaces conditional with two global variables: 4.2 seconds

And just this one new line takes 3.5 seconds
````timeit.timeit('options.get("units_iec")',setup='from bleachbit.Options import options', number=100000)````

However, bytes_to_human on 1000 files, which is more typical than 100,000 files, it is only 0.1 seconds, so maybe not worth optimizing.